edison/src/main/java/edu/illinois/cs/cogcomp/edison/annotators/GazetteerViewGenerator.java

```diff
@@ -97,11 +99,13 @@
 
     private Set<String> ignore;
 
-    public GazetteerViewGenerator(String directory, String viewName) throws Exception {
-        this(directory, true, viewName);
+    private boolean loadFromFromClasspath = false;
+
+    public GazetteerViewGenerator(String directory, String viewName, boolean loadFromFromClasspath) throws Exception {
+        this(directory, true, viewName, loadFromFromClasspath);
     }
 
-    public GazetteerViewGenerator(String directory, boolean gzip, String viewName) throws Exception {
+    public GazetteerViewGenerator(String directory, boolean gzip, String viewName, boolean loadFromFromClasspath) throws Exception {
```

Comment:
I tried to retain the old functionality, by just keeping the old code in place, in case we decide go back to the old usage. In principle we can just use our git history, if that happens. So I can just clean up the classpath-based usages.